File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/youtube/playerResponse/ImpressionEndpointsItem.java
package com.github.kotvertolet.youtubejextractor.models.youtube.playerResponse;

import java.io.Serializable;

public class ImpressionEndpointsItem implements Serializable {

    private FeedbackEndpoint feedbackEndpoint;
    private String clickTrackingParams;

    public FeedbackEndpoint getFeedbackEndpoint() {
        return feedbackEndpoint;
    }

    public void setFeedbackEndpoint(FeedbackEndpoint feedbackEndpoint) {
        this.feedbackEndpoint = feedbackEndpoint;
    }

    public String getClickTrackingParams() {
        return clickTrackingParams;
    }

    public void setClickTrackingParams(String clickTrackingParams) {
        this.clickTrackingParams = clickTrackingParams;
    }

    @Override
    public String toString() {
        return
                "ImpressionEndpointsItem{" +
                        "feedbackEndpoint = '" + feedbackEndpoint + '\'' +
                        ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                        "}";
    }
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/ImpressionEndpointsItem.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class ImpressionEndpointsItem : Parcelable, Serializable {
    @SerializedName("feedbackEndpoint")
    var feedbackEndpoint: FeedbackEndpoint? = null

    @SerializedName("commandMetadata")
    var commandMetadata: CommandMetadata? = null

    @SerializedName("clickTrackingParams")
    var clickTrackingParams: String? = null
    override fun toString(): String {
        return "ImpressionEndpointsItem{" +
                "feedbackEndpoint = '" + feedbackEndpoint + '\'' +
                ",commandMetadata = '" + commandMetadata + '\'' +
                ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                "}"
    }
}